volume-specifications/vs-0000-0010.yml

```diff
@@ -0,0 +1,14 @@
+---
+volume: gbenchvol
```

Comment:
Volume name should be from vc_defn so that we do not have to repeat it.. (mentioning here as well so that we do not miss it)